yocs_joyop/launch/joyop.launch

```
@@ -5,33 +5,41 @@
+	<arg name="more_magic_button"     default="3"/>
+	<arg name="spin_frequency"        default="10"/>
+	<arg name="joy_topic_name"        default="joy"/>
+	<arg name="magic_topic_name"      default="magic"/>
+	<arg name="more_magic_topic_name" default="more_magic"/>
+	<arg name="cmd_vel_topic_name"    default="cmd_vel"/>
+	<arg name="enable_topic_name"     default="enable"/>
+	<arg name="disable_topic_name"    default="disable"/>
  
-  <node pkg="yocs_joyop" type="joyop" name="yocs_joyop" output="screen">
-    <param name="linear_axis"         value="$(arg linear_axis)"/>
-    <param name="angular_axis"        value="$(arg angular_axis)"/>
-    <param name="linear_scale"        value="$(arg linear_scale)"/>
-    <param name="angular_scale"       value="$(arg angular_scale)"/>
-    <param name="deadman_button"      value="$(arg deadman_button)"/>
-    <param name="enable_button"       value="$(arg enable_button)"/>
-    <param name="disable_button"      value="$(arg disable_button)"/>
-    <param name="spin_frequency"      value="$(arg spin_frequency)"/>
-    <remap from="joy"                 to="$(arg joy_topic_name)"/>
-    <remap from="yocs_joyop/cmd_vel"  to="$(arg cmd_vel_topic_name)"/>
-    <remap from="yocs_joyop/enable"   to="$(arg enable_topic_name)"/>
-    <remap from="yocs_joyop/disable"  to="$(arg disable_topic_name)"/>
-  </node>
+    <node pkg="yocs_joyop" type="joyop" name="yocs_joyop" output="screen">
+		<param name="linear_axis"           value="$(arg linear_axis)"/>
+		<param name="angular_axis"          value="$(arg angular_axis)"/>
+		<param name="linear_scale"          value="$(arg linear_scale)"/>
+		<param name="angular_scale"         value="$(arg angular_scale)"/>
+		<param name="deadman_button"        value="$(arg deadman_button)"/>
+		<param name="enable_button"         value="$(arg enable_button)"/>
+		<param name="disable_button"        value="$(arg disable_button)"/>
+		<param name="magic_button"          value="$(arg magic_button"/>
```

This should throw an error. Missing a ")" at the end.